includes/class-wc-taxjar-connection.php

```diff
@@ -61,7 +61,7 @@ private function check_status() {
 			if ( $this->integration->post_or_setting( 'api_token' ) && isset( $body->valid ) && false === $body->valid ) {
 				$description .= '<span style="color: #ff0000;"><strong>';
 				$description .= 'It looks like your API token is invalid.';
-				$description .= sprintf( '<br><a href="%s" target="_blank">Please review your API token.</a>', $this->integration->app_uri . 'account#api-access' );
+				$description .= sprintf( '<br><span style="color: black;">Please attempt to reconnect to TaxJar or </span><a href="%s" target="_blank">review your API token here.</a>', $this->integration->app_uri . 'account#api-access' );
```

Comment:
I think we could remove the `here` wording. I think the `review your API Token` terminology is sufficient. "Click here" type wording in links is generally discouraged.